sdk/communication/Azure.Communication.Identity/README.md

```diff
@@ -152,6 +171,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 <!--[package]: https://www.nuget.org/packages/Azure.Communication.Identity-->
 [product_docs]: https://docs.microsoft.com/azure/communication-services/overview
 [nuget]: https://www.nuget.org/
+[IdentitySDKTurnPrivatePreview]: https://dev.azure.com/azure-sdk/public/_packaging?_a=package&feed=azure-sdk-for-net&package=Azure.Communication.Identity&protocolType=NuGet&version=1.0.0-beta.4&view=versions
```

Comment:
Is this needed? Also the `version=1.0.0-beta.4` is wrong here

Reply:
I wasn't sure what's the best way for this readme to be self-documenting. I wanted to link to the latest released SDK version, but that can't be determined until the PR is merged and pipeline is run.

Reply:
I like that you are trying to have full documentation here. It might be better to explain which NuGet feed should be added.